Robinhood/Robinhood.py

```diff
@@ -1,30 +1,44 @@
-"""Robinhood.py: a collection of utilities for working with Robinhood's Private API"""
-import getpass
-import logging
-import warnings
-from enum import Enum
+"""
+    Robinhood.py: a collection of utilities for working with Robinhood's Private API
+"""
 
-import requests
-import six
+from enum import Enum
 from six.moves.urllib.parse import unquote
 from six.moves.urllib.request import getproxies
 from six.moves import input
 from . import exceptions as RH_exception
 
+import getpass
+import requests
+import six
+
+import logging
+import warnings
+
+
 class Bounds(Enum):
-    """enum for bounds in `historicals` endpoint"""
+    """
+        Enum for bounds in `historicals` endpoint
```

Comment:
adding newline for all docstrings is not required with Google/Napoleon style: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Good point - I just did it to increase readability. Would you like me to go back and fix all of it?

Reply:
Yes, b/c it will make a big mess when someone comes through with `sphinx` to make ReadTheDocs